### PR TITLE
[android] 13426-missing_seqnum 

### PR DESCRIPTION
**Pull request by dominicmarmionswrve**
Fixed a bug where cachedLocalStorage is null for non-init'ed sdk (for push).

https://swrvedev.jira.com/browse/SWRVE-13426

@Sergio-Mira please (thank goodness for unit tests! )

### CODE REVIEW

File: SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java [hunk 1]
```diff
@@ -1663,12 +1664,27 @@ public int getMaxEventsPerFlush() {
 
     @Override
     public synchronized int getNextSequenceNumber() {
-        String id = cachedLocalStorage.getSharedCacheEntry("seqnum");
+        IMemoryLocalStorage memoryLocalStorage = cachedLocalStorage;
+        if(memoryLocalStorage == null) {
```
Comment (Sergio-Mira) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java]: Is this overwritten when the SDK is init'ed?

**Comment by Sergio-Mira** [comment]
🚀

**Comment by dominicmarmionswrve** [comment]
retest me please

**Comment by dominicmarmionswrve** [comment]
@Sergio-Mira sorry, some more changes. 🍻

File: SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java [hunk 2]
```diff
@@ -362,14 +357,6 @@ protected MemoryCachedLocalStorage createCachedLocalStorage() {
         return new MemoryCachedLocalStorage(new MemoryLocalStorage(), null);
     }
 
-    protected ExecutorService createStorageExecutor() {
```
Comment (Sergio-Mira) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java:365]: Does this not break the set of private tests?

**Comment by Sergio-Mira** [comment]
🚀

File: SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java (hunk 2 above)
Comment (dominicmarmionswrve) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java:365]: Something must of used it at one stage, but nothing was using it anymore. So works fine.

**Pull request merged**